### PR TITLE
How to see hidden files

### PR DESCRIPTION
**Pull request by szydan**
Made based on my previous gist https://gist.github.com/szydan/38438470de9bffebd64757bd0859187b

### CODE REVIEW

File: osx/see-all-files.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# See all files in finder
+
+The MacOS Finder application by default is not showing you all files.
```
Comment (emmaxdan) [inline osx/see-all-files.md]: The MacOS Finder application by default is not showing you all your files.

File: osx/see-all-files.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+This is OK if you are not using your computer for programin. Why to see the hidden files, sure there is a reason they are hidden.
```
Comment (emmaxdan) [inline osx/see-all-files.md]: the 2nd sentence doesn't make sense

File: osx/see-all-files.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+But if you are using your computer to program it is handy to always see all the files present in any folder in your system 
+To do it paste the followin command in your terminal
```
Comment (emmaxdan) [inline osx/see-all-files.md]: To do this, paste the following command in your terminal

File: osx/see-all-files.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+```sh
+defaults write com.apple.finder AppleShowAllFiles YES
+```
+
+
+Press enter
+
+Hold the ‘Option/alt’ key, then right click on the Finder icon in the dock and click Relaunch.
+
+From now on you should see all hidden files in any folder
```
Comment (emmaxdan) [inline osx/see-all-files.md]: all the hidden files

File: osx/see-all-files.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+defaults write com.apple.finder AppleShowAllFiles YES
+```
+
+
+Press enter
+
+Hold the ‘Option/alt’ key, then right click on the Finder icon in the dock and click Relaunch.
+
+From now on you should see all hidden files in any folder 
+To check open your home folder in the finder app by typing
```
Comment (emmaxdan) [inline osx/see-all-files.md]: To check, open your home folder in the finder app by typing

File: osx/see-all-files.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+Hold the ‘Option/alt’ key, then right click on the Finder icon in the dock and click Relaunch.
+
+From now on you should see all hidden files in any folder 
+To check open your home folder in the finder app by typing 
+
+```
+open ~/
+```
+
+You will see quite a few new files which starts with "." which you could not se before
```
Comment (emmaxdan) [inline osx/see-all-files.md]: You will see quite a few new files that start with "." which you could not see before

**Review by emmaxdan — CHANGES_REQUESTED** [review]

**Review by emmaxdan — COMMENTED** [review]
i tested it and it works so all you have to do is go over the changes i suggested

File: osx/see-all-files.md (hunk 2 above)
Comment (szydan) [reply osx/see-all-files.md]: @emmaxdan I rewrote it, please check again

**Pull request merged**